packages/vaadin-grid/src/vaadin-grid.js

```diff
@@ -4,10 +4,11 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { beforeNextRender } from '@polymer/polymer/lib/utils/render-status.js';
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
```

Comment:
Let's move the change in this file to a separate PR. Also, let's make this import the first one in the list.